geom/xy_test.go

```diff
@@ -52,3 +53,40 @@ func TestXYUnit(t *testing.T) {
 		})
 	}
 }
+
+func TestXYCross(t *testing.T) {
+	xy := func(x, y float64) geom.XY {
+		return geom.XY{X: x, Y: y}
+	}
+	for i, tc := range []struct {
+		u, v geom.XY
+		want float64
+	}{
+		// Contains zero:
+		{u: xy(1, 1), v: xy(0, 0), want: 0},
+		{u: xy(0.1, 0.1), v: xy(0, 0), want: 0},
+
+		// Not linearly independent:
```

Comment:
Maybe could reword this as `Linearly dependent:`?

Reply:
Good suggestion, altered in 51c7c7327f57a19bf873a4133046c5c16e22e8d5.